### ISSUE TITLE
References to old ISO schemas URL

### ISSUE THREAD

**Issue by rmalyankar** [ISO-TC211/XML#210]
The file gmlWrapperTypes2014.xsd on the standards.iso.org site (home>iso>19115>-3>gmw>1.0) references the schemaLocation "http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19136_Schemas/gml.xsd", but at the time I am writing this, accessing that results in a 404 error, causing schema validation failures. Shouldn't the standards.iso.org server redirect accesses to the correct location?

**Comment by ejbleys** [comment]
Not sure
> standards.iso.org <http://standards.iso.org/> has been deprecated
> 
> Please use https://schemas.isotc211.org <https://schemas.isotc211.org/> 
> 
>  https://schemas.isotc211.org/19115/-3/gmw/1.0/gmlWrapperTypes2014.xsd <https://schemas.isotc211.org/19115/-3/gmw/1.0/gmlWrapperTypes2014.xsd> references http://schemas.opengis.net/gml/3.2.1/gml.xsd <http://schemas.opengis.net/gml/3.2.1/gml.xsd> 

Evert Bleys
4 Tudor Place
HUGHES ACT 2605
Australia
email:  ejbleys@gmail.com
Mob:   +61 (0)411 483 876
Land:  +61 (0)2 6281 1773
Skype: ejbleijs@gmail.com

**Comment by rmalyankar** [comment]
Deprecated it may be, but it should still work, at least by means of a URL redirection.

**Comment by shane-axiom** [comment]
Agreed that the old http://standards.iso.org/ittf/PubliclyAvailableStandards/ links should redirect, as now any XML document using those XSDs in xsi:schemaLocation suddenly fail validation. Not great!

**Comment by MartinePaepen** [comment]
I also encounter this problem now.
Isn't it possible to rederict all the xsd files to the new xsd files?
Now it redirects to an http page but the validation using the xsd files fails.
Which is really a big issue.

**Comment by ejbleys** [comment]
TC211 has no control over any of the domains other than https:…/isotc211.org/

Cheers
E
Evert Bleys
4 Tudor Place
HUGHES ACT 2605
Australia
email:  ***@***.***
Mob:   +61 (0)411 483 876
Land:  +61 (0)2 6281 1773
Skype: ***@***.***

**Comment by MartinePaepen** [comment]
Who is responsible for the http://standards.iso.org/ittf/PubliclyAvailableStandards/  urls?   
or in other words the http:// .../standards.iso.org/ domain?

**Comment by ejbleys** [comment]
That is an interesting question!
e
Evert Bleys
4 Tudor Place HUGHES ACT 2605
Australia
+61 (0)2 62811773
+61 (0)411 483 876
***@***.***
Skype ***@***.***

**Comment by PeterParslow** [comment]
standards.iso.org belongs to the whole ISO organisation, and is managed by the ISO Central Secretariat. TC211 has a "Technical Programme Manager" in ISO/CS who is our point of contact in regards to that domain (and much else).
